frontend/admin/src/js/components/poolCandidate/PoolCandidatesTable.tsx

```diff
@@ -435,18 +523,27 @@ const PoolCandidatesTable: React.FC<{ poolId: string }> = ({ poolId }) => {
     <div data-h2-margin="base(x1, 0)">
       <h2 id="user-table-heading" data-h2-visibility="base(invisible)">
         {intl.formatMessage({
-          defaultMessage: "All Users",
-          id: "VlI1K4",
-          description: "Title for the admin users table",
+          defaultMessage: "All Pool Candidates",
+          id: "z0QI6A",
+          description: "Title for the admin pool candidates table",
         })}
       </h2>
       <TableHeader
         columns={columns}
-        onFilterChange={() => {
-          /* TODO: Implement this later */
-        }}
-        onSearchChange={() => {
-          /* TODO: Implement this later */
+        filterButton={
+          <PoolCandidateTableFilterDialog.Button
+            onSubmit={handlePoolCandidateFilterSubmit}
+          />
+        }
```

Comment:
To make it clearer?

```suggestion
        filterButtonComponent={
          <PoolCandidateTableFilterDialog.Button
            onSubmit={handlePoolCandidateFilterSubmit}
          />
        }
```

Reply:
I like that better too 👍 Fixed by [Make filter button prop name more explicit](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4129/commits/e4382eb36ac45231f297feb11da0cd691ba36f8b).